backend/app/infrastructure/__init__.py

```diff
@@ -16,6 +18,8 @@
     "put": ExceptionHandler.print_stacktrace_on_exception(True, True),
 }
 
+max_time = None
```

Comment:
I think we shouldn't store it here (generaly speaking - https://www.reddit.com/r/learnpython/comments/dn001i/why_are_global_variables_evil/). Maybe let's put it into class Config for instance?

Reply:
Instead of introducing max_time here, I've diretly used `app.settings.config` global variable (and removed set_max_time logic).